### PR TITLE
Resource Details Page

### PR DESCRIPTION
**Pull request by kenjinp**
# Resource Pages!
> as part of "project dismantle https://github.com/BlueBrain/nexus-web/pull/112"

## NEW
- viewing/editing resources
- resource details page (somewhat sparse)
- links from the resource list

## Caveats
- It's missing the metaData, which will be added in a subsequent PR. 
- Because the pinterest route is hard to implement, the resource view will not appear in a modal until I make another PR for that. 
- You cannot edit things like views or schemas. I'm not sure why, I think the API has changed? This must be fixed at the SDK level

### CODE REVIEW

File: src/shared/components/Resources/ResourceDetails.less [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.resource-details {
```
Comment (mdupont) [inline src/shared/components/Resources/ResourceDetails.less]: ???

File: src/shared/components/Resources/ResourceDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+const ResourceDetails: React.FunctionComponent<ResourceViewProps> = props => {
+  const { resource, error, isFetching, onSuccess } = props;
+  const [editing, setEditing] = React.useState(false);
+  const [busy, setFormBusy] = React.useState(isFetching);
+
+  const handleSubmit = async (value: any) => {
+    if (resource) {
+      try {
+        setFormBusy(true);
+        console.log(resource);
```
Comment (mdupont) [inline src/shared/components/Resources/ResourceDetails.tsx]: 🔫

File: src/shared/components/Resources/ResourceDetails.less (hunk 1 above)
Comment (kenjinp) [reply src/shared/components/Resources/ResourceDetails.less]: whoops

File: src/shared/components/Resources/ResourceDetails.tsx (hunk 2 above)
Comment (kenjinp) [reply src/shared/components/Resources/ResourceDetails.tsx]: 💥

**Pull request merged**